standalone/src/PythiaEvtGen_Interface.h

```diff
@@ -68,6 +68,8 @@ class PythiaEvtGen_Interface {
 
   bool             debug;
   std::vector<int> B_ids;
+  std::vector<int> C_ids;
```

Comment:
```suggestion
```
In principle only the `Had_ids` are necessary as member variable here, right? The `B` and `C_ids` seem to be mainly used for initializing that, but nowhere else.